packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/legacy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dedicatedCloud_datacenter_backup_legacy_message_1": "Depuis le {{ date }}, la gamme Veeam Backup a évolué.",
```

Comment:
Why is date passed as a parameter ? Maybe the date should be formatted according to the locale ?